README.md

```diff
@@ -359,6 +363,298 @@ gopkg.in/check.v1
 
 </details>
 
+##### Summary Template
+
+The summary is a template to generate the summary of dependencies' licenses based on the [Golang Template](https://pkg.go.dev/text/template). It includes these variables:
+
+|Name|Type|Example|Description|
+|----|----|-------|-----------|
+|LicenseContent|string|`{{.LicenseContent}}`|The project license content, it's relate to the `header.license.spdx-id` config. |
+|Groups|list structure|`{{ range .Groups }}`|The dependency group, all license is aggregate by the same license [SPDX ID](https://spdx.org/licenses/). |
+|Groups.Name|string|`{{.Name}}`|The [SPDX ID](https://spdx.org/licenses/) of dependency. |
+|Groups.Deps|list structure|`{{ range .Deps }}`|All dependencies, it including all of same [SPDX ID](https://spdx.org/licenses/) dependencies under `Groups.Name`. |
+|Groups.Deps.Name|string|`{{.Name}}`|The name of the dependency. |
+|Groups.Deps.Version|string|`{{.Version}}`|The version of the dependency. |
+|Groups.Deps.LicenseID|string|`{{.LicenseID}}`|The [SPDX ID](https://spdx.org/licenses/) of the dependency license. |
+
+<details>
+<summary>Summary Template Generate</summary>
+
+Summary template content:
+```
+{{.LicenseContent }}
+{{ range .Groups }}
+========================================================================
+{{.Name}} licenses
```

Comment:
Do you think it'd be more specific if we rename `{{.Name}}` to something like `{{.LicenseSpdxID}}` or `{{.LicenseName}}`? Because there is another `{{.Name}}` inside the `{{.Deps}}`,

Reply:
I think it could be better to rename it to `{{.LicenseID}}`? WDYT? 
Let it same with the `{{.LicenseID}}` inside the `{{.Deps}}`. For now, they are the same.

Reply:
> I think it could be better to rename it to `{{.LicenseID}}`? WDYT?

`{{.LicenseID}}` is good to me

Reply:
Updated.